backtracking/sudoku_solve.cpp

```diff
@@ -1,7 +1,5 @@
 #include <iostream>
-using namespace std;
-/// N=9;
-int n = 9;
+#define n 9

```

Comment:
haha.. nice trick to replace ambiguous global variables with macro. but not a good idea either. 😏

Reply:
I'll use `9` instead of using a variable. 😄

Reply:
> I'll use `9` instead of using a variable. 😄

oh, please, no. These are not good coding practices. 
Use a local variable and pass arguments

Reply:
What variable name shall we use? LGTM will consider `n` as a very short name.

Reply:
> What variable name shall we use? LGTM will consider `n` as a very short name.
> That is why I used a macro instead.

I have created a PR on your fork: https://github.com/Panquesito7/C-Plus-Plus/pull/11 explaining how to

Reply:
Fix still not applied

Reply:
To be resolved only after fixing it